package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "scripts": {
+    "pretest": "dotnet fable Fuchu.FableTests --outDir bin",
+    "test": "node --eval \"require('esm')(module)('./bin/Main.js')\" ./bin/Main.js ./bin/Main.js ./bin "
```

Comment:
Not sure if that's the reason, but I usually run the tests like this: `npx mocha ./bin -r esm`